cogs/streams.py

```diff
@@ -367,6 +367,48 @@ def __init__(self, bot):
                 fileIO("data/streams/beam.json", "save", self.beam_streams)
 
             await asyncio.sleep(CHECK_DELAY)
+            
+    @commands.command()
+    async def twitchlist(self):
+        """Lists StreamAlerts turned on for Twitch"""
+        await self.bot.say("Twitch Streams set up for Alerts:")
+        twitempty = 1
+        for item in self.twitch_streams:
+            await self.bot.say(item["NAME"])
+            twitempty = 0
+
+        if twitempty == True:
+            await self.bot.say("No Twitch streams are set to Alert in this channel.")
+
+        fileIO("data/streams/twitch.json", "r", self.twitch_streams)
```

Comment:
ooo inline comments.

what's this and why is it here?

also, propose to change !hitboxlist to !hitlist  :3